src/test/TxTests.h

```diff
@@ -138,6 +138,10 @@ TransactionFramePtr transactionFromOperations(Application& app,
                                               SequenceNumber seq,
                                               std::vector<Operation> const& ops,
                                               uint32_t fee = 0);
+TransactionFramePtr sorobanTransactionFromOperations(
```

Comment:
do we need this separate function given that we have `sorobanTransactionFrameFromOps` already? you could update the existing function to generate desired signatures (also, if you don't need to override signatures too often, you could add the override logic directly into the test that needs it, like we do in other tests):
https://github.com/stellar/stellar-core/blob/af8b65a9686901ec97ea0609823a62951009d8fc/src/herder/test/HerderTests.cpp#L272